<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startupForm</class>
 <widget class="QWidget" name="startupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="recentlyUsedRadioButton">
     <property name="text">
      <string>Otwórz ostatnio używany słownik</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="selectedRadioButton">
     <property name="text">
      <string>Otwórz wybrany słownik</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="selectedComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectedRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
